src/temp/ProjectMapTabs/SingleTab.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './ProjectMapTabs.module.scss';
+
+interface SingleTabProps {
+  icon: React.JSX.Element;
+  title: string;
+  isSelected: boolean;
+}
+
+const SingleTab = ({ icon, title, isSelected }: SingleTabProps) => {
+  return (
+    <>
+      <button
+        className={`${styles.option} ${isSelected ? styles.selected : ''}`}
```

Comment:
Can you rename the `option` class to `tabOption` or `singleTab`? If we follow the convention of keeping the classname and the component name the same, it enhances css readability.